tests/includes/activity/class-test-activity.php

```diff
@@ -132,4 +133,40 @@ public function test_activity_object_id() {
 
 		$this->assertTrue( str_starts_with( $activity->get_id(), 'https://example.com/author/123#activity-update-' ) );
 	}
+
+	/**
+	 * Test activity object.
+	 */
+	public function test_activity_object_in_reply_to() {
+		// Create user with `activitypub` capabilities.
+		$user_id = self::factory()->user->create(
+			array(
+				'role' => 'author',
+			)
+		);
+
+		// Only send minimal data.
+		$activity_object = array(
+			'id'     => 'https://example.com/post/123',
+			'type'   => 'Follow',
+			'actor'  => 'https://example.com/author/123',
+			'object' => 'https://example.com/post/123',
+			'to'     => array( 'https://example.com/author/123' ),
+		);
+
+		$activity = new Activity();
+		$activity->set_type( 'Accept' );
+		$activity->set_actor( Actors::get_by_id( $user_id )->get_id() );
+		$activity->set_object( $activity_object );
+
+		$this->assertEquals( 'https://example.com/author/123', $activity->get_to()[0] );
```

Comment:
I think this is fine, no need to change. For future tests, where you might not have control over the array that you want to test, `assertContains` might be helpful.